docs/user_guide/security/serialization.rst

```diff
@@ -5,4 +5,226 @@ Message Serialization
 NVFLARE uses a secure mechanism called FOBS (Flare OBject Serializer) for message serialization and
 deserialization when exchanging data between the server and clients.
 
-See :github_nvflare_link:`README <nvflare/fuel/utils/fobs/README.rst>` for usage guidelines.
+
+Flare Object Serializer (FOBS)
+==============================
+
+
+Overview
+--------
+
+FOBS is a drop-in replacement for Pickle for security purposes. It uses **MessagePack** to
+serialize objects.
+
+FOBS sacrifices convenience for security. With Pickle, most objects are supported
+automatically using introspection. To serialize an object using FOBS, a **Decomposer**
+must be registered for the class. A few decomposers for commonly used classes are
+pre-registered with the module.
+
+FOBS supports enum types by registering decomposers automatically for all classes that
+are subclasses of :code:`Enum`.
+
```

Comment:
Should add a paragraph here,

FOBS treats all other classes as dataclass by registering a generic decomposer for dataclasses. Dataclass is a class whose constructor only changes the state of the object without side-effects. Side-effects include changing global variables, creating network connection, files etc.

Reply:
thanks, good suggestion, added